src/frontend/src/components/public/PIASideNav/pia-form-sideNav-pages.ts

```diff
@@ -217,24 +219,69 @@ if it is ++ or -- operater navigate to the previous or next tab
         },
       },
     },
-    {
-      id: 10,
-      label: 'Additional risks',
-      link: buildDynamicPath(
-        isEditMode
-          ? routes.PIA_ADDITIONAL_RISKS_EDIT
-          : routes.PIA_ADDITIONAL_RISKS_VIEW,
-        {
-          id: pia?.id,
-        },
-      ),
-      enable: showPostIntakeTabs,
-      state: {
-        prev: {
-          condition: true,
-          action: -1,
-        },
-      },
-    },
+    ...(!isMPORole()
+      ? [
+          {
+            id: 10,
+            label: 'Additional risks',
+            link: buildDynamicPath(
+              isEditMode
+                ? routes.PIA_ADDITIONAL_RISKS_EDIT
+                : routes.PIA_ADDITIONAL_RISKS_VIEW,
+              {
+                id: pia?.id,
+              },
+            ),
+            enable: showPostIntakeTabs,
+            state: {
+              prev: {
+                condition: true,
+                action: -1,
+              },
+            },
+          },
+        ]
+      : [
+          {
+            id: 10,
+            label: 'Additional risks',
+            link: buildDynamicPath(
+              isEditMode
+                ? routes.PIA_ADDITIONAL_RISKS_EDIT
+                : routes.PIA_ADDITIONAL_RISKS_VIEW,
+              {
+                id: pia?.id,
+              },
+            ),
+            enable: showPostIntakeTabs,
+            state: {
+              next: {
+                condition: true,
+                action: +1,
+              },
```

Comment:
Since it's just the `next` that is changing, can we add condition here instead of the whole additional risk tab?

Reply:
Sure, I just thought that it was more readable this way (the change that is.)